v2/cmd/launch/main.go

```diff
@@ -55,3 +56,11 @@ func main() {
 		glog.Exitf("Failed to execute component: %v", err)
 	}
 }
+
+func init() {
+	// Change default logging level to facilitate debugging
+	flag.Set("logtostderr", "true")
+	flag.Set("stderrthreshold", "WARNING")
+	// Uncomment the following to enable INFO level logging.
+	// flag.Set("stderrthreshold", "INFO")
```

Comment:
Instead of this line, consider adding a comment above to instruct user to change "WARNING" to "INFO"